sdk/src/main/java/cryptohopper/android/sdk/CryptohopperHopper.kt

```diff
@@ -849,6 +850,27 @@ class CryptohopperHopper {
             })
         }
 
+
+        /// Hold Multiple Positions
+        ///
+        /// - Parameter hopperId:  (required) Hopper Id
+        /// - Parameter positionIds:  (required) Position Ids
+        fun holdMultipleHopperPositions(
+            hopperId: String,
+            positionIds: List<Int>,
+            callback: (String?, HopperAPIError?) -> Unit
+        ) {
+            HopperAPIHoldMultiplePositionsRequest(
+                hopperId,
+                positionIds
+            ).request<HopperCommonMessageResponse>({ data ->
+                callback(data.data, null)
```

Comment:
I am not sure this is a valid check or not , but in order to understand if one position is actually hold or not you also need to call get position by id endpoint to check if it's hold or not